app/controllers/volunteer_ops_controller.rb

```diff
@@ -75,6 +87,27 @@ def build_map_markers(organisations)
     end
   end
 
+  # This needs to be a helper method for the do-it API
+  def collect_all_items (host, href, orgs)
+    n=1
+    while href do
+      url = host + href
+      response = HTTParty.get(url)
+      if response.body && response.body != '[]'
+        respItems = JSON.parse(response.body)["data"]["items"]
+        respItems.each do |item|
+          org = OpenStruct.new(latitude: item["lat"], longitude: item["lng"], name: item["title"], id: n,
+                                description: item["description"], opportunity_id: item["id"],
```

Comment:
Metrics/LineLength: Line is too long. [93/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)